converter/output/rita/db.go

```diff
@@ -0,0 +1,140 @@
+package rita
```

Comment:
This file encapsulates a lot of the logic that was previously spread between the db manager (OutputDB) and the session writers.